documentation/configure.markdown

```diff
@@ -914,3 +916,53 @@ Enabling default pretty-printing is not recommended in production because it
 incurs a penalty in data transfer speed and size. Users may override this
 setting on a per-query basis by supplying a `?pretty=` parameter in the URL,
 valued `true` or `false`.
+
+`[sync]` settings (Puppet Enterprise only)
+-----
+
+The `[sync]` section of the PuppetDB configuration file is used to configure
+synchronization, for a high-availability system. See
+the [HA configuration guide][ha] for complete system configuration instructions.
+
+### `remotes`
+
+The `remotes` configuration key indicates that PuppetDB should poll a remote
+PuppetDB server for changes. When it finds changed or updated records on that
+server, it will download the records and submit them to the local command queue.
+
+In the configuration file, you specify a `remote` for each server you want to
+pull data from. (It is perfectly reasonable, and expected, for two servers to
+pull data from each other.) For each remote, you must provide:
```

Comment:
We should either remove the parens, or stick the parenthetical in the preceding sentence